frontend/src/metabase/visualizations/shared/components/RowChart/RowChart.tsx

```diff
@@ -89,7 +92,7 @@ export const RowChart = <TDatum,>({
 
   xScaleType = "linear",
 
-  measureText,
+  measureText: measureTextSize,
```

Comment:
What do you think about keeping `measureText` as before i.e. only measures width? And create a new function called `measureTextDimension` that returns `width` and `height` for https://github.com/metabase/metabase/blob/3b08aaf4cd215b2426020131d703f369f1a2b080/frontend/src/metabase/visualizations/components/ScalarValue/utils.ts#L28-L32

This way we don't have different APIs for `measureText` which could create confusion and cause name conflicts just like this.